framework/include/base/MooseObjectWarehouseBase.h

```diff
@@ -539,6 +550,56 @@ MooseObjectWarehouseBase<T>::updateVariableDependencyHelper(std::set<MooseVariab
   }
 }
 
+template<typename T>
+void
+MooseObjectWarehouseBase<T>::updateMatPropDependency(std::set<unsigned int> & needed_mat_props, THREAD_ID tid /* = 0*/) const
```

Comment:
An extra space between `tid` and `/*`. The same in `updateBlockMatPropDependency` below.